NuRadioMC/test/atmospheric_Aeff/1e18eV/test_build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+NuRadioMC/test/atmospheric_Aeff/1e18eV/T01generate_event_list.py
+
+NuRadioMC/test/atmospheric_Aeff/1e18eV/T02RunSimulation.py 1e18_full.hdf5  ../dipole_100m.json ../config.yaml output.hdf5 output.nur
+
+NuRadioMC/test/atmospheric_Aeff/1e18eV/T03check_output.py
+
+rm NuRadioMC/test/atmospheric_Aeff/1e18eV/tables/*txt
```

Comment:
why is there a run and a test_build script? I think we only need one.

Reply:
No, we don't. That's what you can find on the Veff example.

Reply:
Changed.